src/api/expense.controller.js

```diff
@@ -0,0 +1,94 @@
+const { expenseService } = require('../services/expense.service.js');
+const { userService } = require('../services/user.service.js');
+
+const getExpenses = async (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const categoryList = categories ? categories.split(',') : [];
+
+  const expenses = await expenseService.getExpenses({
+    userId,
+    categoryList,
+    from,
+    to,
+  });
+
+  res.json(expenses);
+};
+
+const createExpense = async (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
```

Comment:
It might be beneficial to validate the `spentAt` field to ensure it is a valid date. This can help prevent errors when processing the expense data.